config/jest-config-carbon/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Change Log
```

Comment:
Make sure to remove any CHANGELOGs that are generated

Reply:
yep I saw that 🤕